### PR TITLE
Add README for mta

### PR DESCRIPTION
**Pull request by rgolangh**
FLPATH-769

Signed-off-by: Roy Golan <rgolan@redhat.com>

### CODE REVIEW

**Review by rgolangh — COMMENTED** [review]
@y-first PTAL

File: mta/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# MTA - migration analysis workflow
+
+# Synopsis
+This workflow is an assessment workflow type, that invokes an application analysis workflow using MTA[1]
+and optionally returns the [move2kube](./m2k) workflow reference, to run next if the analysis is considered to be successful.
```
Comment (y-first) [inline mta/README.md:5]: 'Optionally' is misleading. The user can't currently choose whether to return the M2K workflow or not - it's linked in the Results no matter what.

File: mta/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+Users are encouraged to use this workflow as self-service alternative for interacting with the MTA UI. Instead of running
+a mass-migration of project from a managed place, the project stakeholders can use this (or automation) to regularly check
+the cloud-readiness compatibility of their code.
+
+# Inputs
+- `repositoryUrl` [mandatory] - the git repo url to examine
+- `backstageUserOrGroup` [optional] - the backstage user or group to send backstage notification with the analysis results
+
+# Output
+1. On completion the workflow returns an options[2] structure in the exit state of the flow(also named variables in SonataFlow)
```
Comment (y-first) [inline mta/README.md:16]: nit - should have a space between flow and parentheses.

File: mta/README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+linking to the [move2kube][3] workflow that will generate k8s manifests for container deployment.
+1. When the workflow completes there should be a report link on the exit state of the flow (also named variables in SonataFlow)
```
Comment (y-first) [inline mta/README.md:18]: The report link is currently only available through the notifications. There isn't another way for the user to see it.

Comment (rgolangh) [reply mta/README.md:18]: It was written to the context of the workflow, and was in the 'variables' section in the workflow instance run page. 
I think the section of that writes the output option overwrote that - you can open a bug for it .

@anludke was it necessary to remove "+=" in #37 so the rendering in the ui works?

File: mta/README.md (hunk 1 above)
Comment (rgolangh) [reply mta/README.md:5]: Ack. will update

File: mta/README.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+If `backstageUserOrGroup` is set then the link is also attached to a backstage notification.
```
[the remark this answers is not in the document]
Comment (rgolangh) [reply mta/README.md:19]: https://github.com/parodos-dev/serverless-workflows/blob/ec5b121a1ae5925f04022d4262471afe49d8f75a/mta/mta.sw.yaml#L153

File: mta/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (rgolangh) [reply mta/README.md]: linked to the json schema of the output

File: mta/README.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+Currently this is working with MTA version 6.2.x and in the future 7.x version the report link will be removed or will be made
```
[the remark this answers is not in the document]
Comment (rgolangh) [reply mta/README.md:20]: I think its going to removed

**Comment by rgolangh** [comment]
According the MTA 7 user interface[1] guide there will still be the option to download 
html report, but it maybe be disabled by admins, and will require explicitly enabling it.

I'm not sure how critical the report is for users and to the best of my understanding 
its more important we would know how to anaylyze that report and conditionally offer move2kube.

See this for more info in https://issues.redhat.com/browse/FLPATH-941

[1] https://access.redhat.com/documentation/en-us/migration_toolkit_for_applications/7.0/html-single/user_interface_guide/index

File: mta/README.md (hunk 3 above)
Comment (rgolangh) [reply mta/README.md]: 'variables' was renamed to 'results' I see

Comment (y-first) [reply mta/README.md]: Created bug for this: https://issues.redhat.com/browse/FLPATH-1041

**Comment by rgolangh** [comment]
ping?

**Review by masayag — APPROVED** [review]

**Pull request merged**